source/configuration_manual/mail_location/index.rst

```diff
@@ -83,17 +83,27 @@ Key                Value Description
                      The mapping between mailbox names and GUIDs exists in
                      ``dovecot.list.index*`` files.
 
+``KEEP-NOSELECT``  Do not automatically delete ``\NoSelect`` mailboxes that have
+                   no children. This is the inverse of ``NO-NOSELECT``.
+
+                   .. versionadded:: v2.4.0;v3.0.0
+
 ``NO-NOSELECT``    Automatically delete any ``\NoSelect`` mailboxes that have
                    no children. These mailboxes are sometimes confusing to
                    users. Also if a ``\NoSelect`` mailbox is attempted to be
                    created with ``CREATE box/``, it's created as selectable
-                   mailbox instead. (``LAYOUT=Maildir++`` always behaves this
-                   same way.)
+                   mailbox instead.
+
+                   NO-NOSELECT is the default behavior. Note that
+                   ``LAYOUT=Maildir++`` always behaves this same way regardless
+                   of the setting.
```

Comment:
Note that  ``LAYOUT=Maildir++`` always behaves this same way regardless of the setting.

Note that  ``LAYOUT=Maildir++`` always behaves the same way regardless of the setting. 

this -> the ?

Reply:
I used this instead of the to clarify that it behaves THIS way not just a random same way. Maybe I could remove 'same' instead ?